avm/res/app/container-app/main.bicep

```
@@ -318,6 +321,19 @@ resource containerApp_roleAssignments 'Microsoft.Authorization/roleAssignments@2
   }
 ]
 
+module containerAppAuthConfigs './auth-configs/main.bicep' = if (!empty(authConfigs)) {
+  name: '${uniqueString(deployment().name, location)}-auth-configs'
```

```suggestion
module containerAppAuthConfigs './auth-config/main.bicep' = if (!empty(authConfig)) {
  name: '${uniqueString(deployment().name, location)}-auth-config'
```

A few things to note
- Folders must be singular
- Deployment Names etc. if only one instance is/can be deployed ideally are also singular. Would be different, for example, for `roleAssignments`

---

Sure thing, I was wondering about that one since the resource provider itself is `authConfigs` but will try to remember that for future changes.

---

Much appreciated @oZakari 🚀 
However, the folder name also must be changed. I believe the module should currently show an error

---

Fixed!